rfp-proposals/community-e-europe.md

```diff
@@ -0,0 +1,77 @@
+# RFP Proposal: `Community Eastern Europe`
+
+**Name of Project:**
+
+**Link to RFP:** <a href="https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-community-meetups.md">rfp-community-meetups</a>
+
+**RFP Category:** community
+
+**Proposer:** <a href="https://github.com/AndreiD">AndreiD</a>
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** Yes
+
+# Project Description
+
+
+Tech meetups & workshops for coders, CTO's, startups & techies. Learn from one another and collaborate over the usual free pizza & beer.
+
+
+## Deliverables
+
+Ongoing meetups & workshops in our own venue (Sofia Bulgaria), then we plan to use https://bucharest.techhub.com/ (Bucharest) or  CITY College (Thessaloniki, Greece)
+Questions are approached technically from a computer science point of view, mathematically, or economically.
+Economics plays a big role and we invite economic experts to participate and speak.
+We also invite the most interesting companies within these fields to gather relevant experts and facilitate debates.
+
+## Development Roadmap
+
+- Tickets: $0. We don't plan to sell tickets for this community events.
+- Confirmed Venue (we confirm our venue in Sofia, it's owned by us)
+- Each meetup is recorded & photographed. if anyone doesn't want to appear in photos they can put a yellow sticker on their clothes.
+- We can handle 20-30 people easily
+- Funding: we will try to get the venues for free (the one in Sofia is free as it's ours)
+- Quality Pizza (not Dominos!) & quality beer will likely cost ~50-100 USD per event depending on how many people decide to participate
+- We can also make t-shirts with filecoin logo. A quality t-shirt with a printed logo is around $10, we can also get super-quality t-shirts with $15. for 20 participants that's $200 in budget.
+- $ for marketing the event until places are filled
+- we do it in a not for profit way, all the funds that remain will go into the next meetup budget.
+
+
+## Total Budget Requested
+
+- TBD
+
+## Maintenance and Upgrade Plans
+
+- N/A
+
+# Team
+
+## Team Members
+
+- Andrei Dan
+- Stoyan Antov
+- position opened for anyone else in SE Europe that wants to join efforts
+
+## Team Member LinkedIn Profiles
+
+- I don't use LinkedIn so much
+- https://www.linkedin.com/in/stoyan-antov/
+
+
+## Team Website
+
+https://avid.bg/  and https://www.facebook.com/girlscodebg/
+
+## Relevant Experience
+
+We organized meetups & workshops. Also paid lessons in programming.
+- our last meetup that we organized (Django Girls) has around $500 in budget. 40 people (well women) attended
+see https://djangogirls.org/sofia1/  or photos here -> https://www.facebook.com/pg/girlscodebg/photos/?tab=album&album_id=1688256924819538
```

Comment:
Thanks for the link! Was the last meetup in 2017 or more recently?

Reply:
we constantly organize meetups, but this was more exotic. 2017 November.